src/block/countdown/edit.js

```diff
@@ -0,0 +1,511 @@
+										attribute="minutesLeft"
+									/>
+									<AdvancedRangeControl
+										label={ __( 'Seconds', i18n ) }
+										min={ 0 }
+										max={ 59 }
+										value={ attributes.secondsLeft }
+										attribute="secondsLeft"
+									/>
+									<AdvancedRangeControl
+										label={ __( 'Restart Countdown After no. of Hours', i18n ) }
+										min={ 0 }
+										maxSlider={ 10 }
+										value={ attributes.restartInterval }
+										attribute="restartInterval"
+									/>
+								</Fragment>
+							) }
+							<AdvancedToggleControl
+								label={ __( 'Enable Double Digit', i18n ) }
+								attribute="isDoubleDigitHidden"
+								defaultValue={ false }
+							/>
+							<AdvancedRangeControl
+								label={ __( 'Box Gap', i18n ) }
+								min={ 0 }
+								sliderMax={ 50 }
+								attribute="boxGap"
+								placeholder="16"
+							/>
+							<AdvancedRangeControl
+								label={ __( 'Label Top Margin', i18n ) }
+								min={ 0 }
+								sliderMax={ 50 }
+								attribute="labelMarginTop"
+								placeholder="8"
+							/>
+						</PanelAdvancedSettings>
+					</InspectorStyleControls>
+					<Typography.InspectorControls
+						label={ __( 'Digits', i18n ) }
+						attrNameTemplate="digit%s"
+						hasTextTag={ false }
+						hasTextContent={ false }
+						initialOpen={ false }
+					/>
+					<Typography.InspectorControls
+						label={ __( 'Labels', i18n ) }
+						attrNameTemplate="label%s"
+						hasTextTag={ false }
+						hasTextContent={ false }
+						initialOpen={ false }
+					/>
+					<Divider.InspectorControls />
+					<BlockDiv.InspectorControls />
+					<Advanced.InspectorControls />
+					<Transform.InspectorControls />
+					<ContainerDiv.InspectorControls sizeSelector=".stk-block-countdown__content" />
+					{ attributes.actionOnExpiration === 'showMessage' &&
+						<Typography.InspectorControls
+							label={ __( 'Expired Message', i18n ) }
+							attrNameTemplate="message%s"
+							hasTextTag={ true }
+							hasTextContent={ true }
+							initialOpen={ false }
+						/> }
+					<EffectsAnimations.InspectorControls />
+					<CustomAttributes.InspectorControls />
+					<CustomCSS.InspectorControls mainBlockClass="stk-block-countdown" />
+					<Responsive.InspectorControls />
+					<ConditionalDisplay.InspectorControls />
+				</>
+			) }
+
+			<CountdownStyles
+				version={ VERSION }
+				blockState={ props.blockState }
+				clientId={ clientId }
+			/>
+			<CustomCSS mainBlockClass="stk-block-countdown" />
+
+			<BlockDiv
+				blockHoverClass={ props.blockHoverClass }
+				clientId={ props.clientId }
+				attributes={ props.attributes }
+				className={ blockClassNames }
+			>
+				<div className="stk-block-countdown__container">
+					{ attributes.dayShow &&
+						<ContainerDiv className={ contentClassNames }>
+							<div className="stk-block-countdown__container-wrapper">
+								<CountdownNumber
+									className={ dayDigitClassNames }
+									type={ 'days' }
```

Comment:
Adjust similar items to string constants
```suggestion
									type="days"
```